_includes/neurips/2019/presenters.neurips.2019.html

```diff
@@ -0,0 +1,146 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Speakers</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <div class="grid">
+        {% assign featuredItems = site.tags.oral-neurips2019% | sort: "featuredOrder" %} 
+        {% for post in featuredItems %}
+            <div class="cell speakers-item">
+                <a href="{{post.website}}" target="_blank" class="speakers-link">
+                    <div class="speakers-hover">
+                        <div class="speakers-hover-content">
+                            <p>{{post.topic}}</p>
+                            <i class="fa fa-plus fa-3x"></i>
+                        </div>
+                    </div>
+                    <img src="/assets/img/NIPS/Speakers/{{ post.img }}" class="responsive-image img-centered desaturate" alt="">
+                </a>
+                <div class="speakers-caption">
+                    <h4>{{ post.name }}</h4>
+                    <p class="text-muted">{{ post.title }}</p>
+                </div>
+            </div>
+        {% endfor %}
+        </div>
+    </div>
+</section>
+<section>
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Posters</h2>
+                <h3 class="section-subheading text-muted"></h3>
+            </div>
+        </div>
+        <ul>
+            	<li>Dennis H Núñez Fernández (Universidad Nacional de Ingeniería)* "Development of a hand pose recognition system on an embedded computer using CNNs</li>
```

Comment:
Poster presenters list now begins with a mixture of first names or titles, please arrange in the following format: 

_Name, Institution, Title of Paper_

In ascending alphabetical order of presenter's first name.